bin/init-OpenData.sh

```diff
@@ -30,5 +30,7 @@ if (ckan_cli datarequests --help); then
     fi
 fi
 
-#Initialise the Harvester database tables
-ckan_cli harvester initdb
+#Initialise the Harvester database tables if applicable
+if (ckan_cli harvester initdb --help); then
```

Comment:
how does this help? can you explain

Reply:
Latest upstream Harvester (which we have merged into `develop`) no longer has an initdb command.

Reply:
could you update comment to the version this is supporting then.